lib/internal/streams/buffer_list.js

```diff
@@ -2,14 +2,16 @@
 
 function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
 
-var Buffer = require('buffer').Buffer;
-var util = require('util');
+var _require = require('buffer'),
+    Buffer = _require.Buffer;
 
-var _require = require('util'),
-    inspect = _require.inspect;
+var _require2 = require('util'),
+    inspect = _require2.inspect;
+
+var custom = inspect && custom || Symbol('useless');
```

Comment:
Instead of using `Symbol('useless')` fall back to `'inspect'`. That way old Node versions should work properly.